src/main/java/net/snowflake/ingest/streaming/internal/OpenChannelResponse.java

```diff
@@ -21,6 +21,7 @@ class OpenChannelResponse extends StreamingIngestResponse {
   private List<ColumnMetadata> tableColumns;
   private String encryptionKey;
   private Long encryptionKeyId;
+  private FileLocationInfo icebergLocation;
```

Comment:
Let's make it more generic, something like external volume location?

Reply:
Matching [server side code](https://github.com/snowflakedb/snowflake/blob/62ed29fc77a9302b62f474e40f283c6c42c4a060/GlobalServices/src/main/java/com/snowflake/resources/streaming/ChannelOpenResponse.java#L127). Should we change both?

cc @sfc-gh-hmadan

Reply:
i believe you can update the name but keep the json property?

Reply:
Name updated.